cmd/entrypoint/runner.go

```diff
@@ -42,6 +43,9 @@ func (rr *realRunner) Run(args ...string) error {
 
 	// Start defined command
 	if err := cmd.Start(); err != nil {
+		if ctx.Err() == context.DeadlineExceeded {
```

Comment:
What error does cmd.Start return when the context's deadline is exceeded? Can we just return that? Having to check for ctx.Err() in each of these is cumbersome and we're likely to forget one in future changes.

Reply:
cmd.Start() returns some generic error about the command being killed, this could be for any reason. Therefore we need to check ctx.Err() to know whether this was due to exceeding the timeout so we can have appropriate error reporting.